editor/components/block-list/block-draggable.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Draggable } from '@wordpress/components';
+
+function BlockDraggable( { rootUID, index, uid, layout, isDragging, ...props } ) {
+	const blockDragInsetClassName = classnames( 'editor-block-list__block-draggable', {
```

Comment:
Does the variable name need to be so intricate? The name itself is not very clear to me in what context it's providing.

Reply:
I'm guessing just `draggableClassname` at this point should suffice.

Reply:
Or just `className` or `classes`. "Draggable" can already be assumed to be known by the file / component name.

Reply:
Ah yes, this is no longer in `block.js`. Sorry, many changes.